dask_cudf/io/csv.py

```diff
@@ -18,10 +20,21 @@ def read_csv(path, chunksize="256 MiB", **kwargs):
         path, tokenize, **kwargs
     )  # TODO: get last modified time
 
+    if chunksize is None:
+        return read_csv_with_compression(path, **kwargs)
+
+    compression = kwargs.pop('compression', False)
     meta = cudf.read_csv(filenames[0], **kwargs)
     dsk = {}
     i = 0
     dtypes = meta.dtypes.values
+
+    if chunksize and compression:
+        warn("Warning %s compression does not support breaking apart files\n"
+            "Please ensure that each individual file can fit in memory and\n"
+            "use the keyword ``blocksize=None to remove this message``\n"
+            "Setting ``chunksize=(size of file)``" % compression)
```

Comment:
You'll want to add a `chunksize=None` assignment and then move this conditional above the `if chunksize is None` block.